alerts/geomodel_alert.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
```

Comment:
I think it seems a bit weird to have `_alert` be in the filename, but I think I'd agree with you renaming it away from geomodel (the same as your folder), what about something like `geomodel_location`, thoughts?